experimental/framework/frs-client/src/interfaces.ts

```diff
@@ -26,10 +31,20 @@ interface FrsConnectionConfigWithKey extends FrsConnectionConfigBase {
 
 interface FrsConnectionConfigWithTokenProvider extends FrsConnectionConfigBase {
     type: "tokenProvider";
-    tokenProvider: ITokenProvider
+    tokenProvider: ITokenProvider;
```

Comment:
Side nit: It doesn't look like we exporting the ITokenProvider interface anywhere.

Reply:
Re-exported now from frs-client package